src/routes/FullPageModal/FullPageModal.js

```diff
@@ -28,7 +28,7 @@ export default function FullPageModal ({
 
   if (mobileSettingsLayout) {
     return (
-      <div styleName='modalMobileSettings'>
+      <div styleName='modal--modal-settings-layout'>
```

Comment:
ah the everlasting battle between camels and dashed continues... reasons for a preference for dashes?

Reply:
indeed. SCSS/SASS is is essential a CSS format and by historical convention dashes and underscores were a thing. We started by sticking with that convention and I prefer it as it delineates between imported CSS styles vs Javascript objects styles (which we may also use or end-up switching to at times). I haven't done a recent review of this in Evo but I suspect that the majority of the CSS classes are named using dashes and underscores. It works for me, and I care very little until it's consistent and we've all decided so happy to go either way. Leaving it for now.